app/src/main/java/com/cornellappdev/uplift/ui/screens/MainReminderScreen.kt

```diff
@@ -0,0 +1,113 @@
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavHostController
+import com.cornellappdev.uplift.models.LocationRepository
+import com.cornellappdev.uplift.networking.ApiResponse
+import com.cornellappdev.uplift.ui.screens.subscreens.MainError
+import com.cornellappdev.uplift.ui.screens.subscreens.MainLoaded
+import com.cornellappdev.uplift.ui.screens.subscreens.MainLoading
+import com.cornellappdev.uplift.ui.viewmodels.ClassDetailViewModel
+import com.cornellappdev.uplift.ui.viewmodels.GymDetailViewModel
+import com.cornellappdev.uplift.ui.viewmodels.HomeViewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.rememberMultiplePermissionsState
+import com.valentinilk.shimmer.Shimmer
+import kotlinx.coroutines.delay
+import android.util.Log
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarColors
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.util.GRAY00
+import com.cornellappdev.uplift.util.GRAY01
+import com.cornellappdev.uplift.util.LIGHT_GRAY
+import com.cornellappdev.uplift.util.montserratFamily
+import com.google.accompanist.systemuicontroller.SystemUiController
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+/**
+ * The home page of Uplift.
+ */
+@OptIn(ExperimentalPermissionsApi::class, ExperimentalMaterial3Api::class)
+@Composable
+fun MainReminderScreen() {
+    val systemUiController: SystemUiController = rememberSystemUiController()
+
+    systemUiController.isStatusBarVisible = false
+    systemUiController.isNavigationBarVisible = false // Navigation bar
+    systemUiController.isSystemBarsVisible = false
+
+    Scaffold(topBar = {
+        TopAppBar(
+            title =  {Text(
+                text = "Reminders",
```

Comment:
Also a bit nitpicky, but I think the Reminders text should be centered and a little smaller based on the figma

Reply:
Yeah, definitely some major discrepancies here with Figma and the header.

![image](https://github.com/user-attachments/assets/3aedd0ad-24ba-46c7-9f90-4c7c7593b1db)

Some main points:
- Arrow icon on the left seems too large and not thick enough? (not 100% sure)
- Reminders not centered
- Reminders definitely not the right text style. Here's what the figma says:

```kotlin
// Header/H2
style = TextStyle(
  fontSize = 16.sp,
  lineHeight = 20.sp,
  fontFamily = FontFamily(Font(R.font.montserrat)),
  fontWeight = FontWeight(700),
  color = Color(0xFF222222),
)
```

Since this is given a reusable `header2` name, you should add this as a reusable style. Check resell's [TextStyle ](https://github.com/cuappdev/resell-android/blob/main/app/src/main/java/com/cornellappdev/resell/android/ui/theme/TextStyle.kt)for how I did this there.

Reply:
Plz make sure the other text on this screen aligns with Figma! Screenshot's a little too low res to tell.